cmd/main.go

```diff
@@ -98,6 +101,10 @@ func init() {
 
 	utilruntime.Must(everestv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(vmv1beta1.AddToScheme(scheme))
+
+	utilruntime.Must(pgv2.SchemeBuilder.AddToScheme(scheme))
+	utilruntime.Must(psmdbv1.SchemeBuilder.AddToScheme(scheme))
+	utilruntime.Must(pxcv1.SchemeBuilder.AddToScheme(scheme))
```

Comment:
will it work if initially any of the operators have not been installed?

Reply:
Yes, the scheme only holds a mapping between the APIs/GVK and the Go structs to facilitate encoding/decoding from the Kube API server. It does not talk to Kubernetes